osf/utils/migrations.py

```diff
@@ -390,6 +390,20 @@ def create_schema_blocks_for_question(state, rs, question, sub=False):
         split_options_into_blocks(state, rs, question, schema_block_group_key)
 
 
+def map_schema_to_schemablocks(rs, state=None):
```

Comment:
Pulled this out of the larger function to resuse in the RegistrationSchemaFactory .